apps/dashboard/app/apps/ood_app_link.rb

```diff
@@ -23,5 +23,30 @@ def initialize(config = {})
   def new_tab?
     @new_tab
   end
+
+  def to_h
+    instance_variables.each_with_object({}) do |var, hash|
+      hash[var.to_s.gsub('@', '').to_sym] = instance_variable_get(var)
+    end
+  end
+
+  # make an OodAppLink look like an OodApp
+  class AppDecorator < SimpleDelegator
```

Comment:
This could probably be renamed, maybe `LinkRecategorizer`?  

Also I'm wondering if upper layers should call it directly or if we should have something like this. I'm sort of ambivalent on it, really just thinking out loud.
```ruby
def recategorize(...)
  AppDecorator.new(self, ...)
end

Reply:
I like it.
As OodAppLink does not have a category, should we call it `LinkCategorizer` and :
```ruby
def categorize(...)
  LinkCategorizer.new(self, ...)
end
```

Reply:
thanks!